### ISSUE TITLE
comparison to pymmcore

### ISSUE THREAD

**Issue by ianhi** [micro-manager/pycro-manager#132]
Hi,

Would it be possible to add a brief comparison of pycromanager to pymmcore to either or both of the readme and the docs? This would help reduce the mental burden of choice for someone like me stumbling across both and trying to decide which route to follow. (In my case it was pycromanager bc there are docs :smile: ) but I think  section structured something like this could be helpful:

Use pycromanager if:
- use case one
- use case two

Use pymmcore if:
- use case one
- use case two

**Comment by henrypinkard** [comment]
Not sure if this is entirely figured out yet. pymmcore is older, no docs, no GUI. Potentially faster, but we're working on that (https://github.com/micro-manager/pycro-manager/issues/90). If that pans out, the only reason to use pymmcore would be that you want to use something lighter weight with no GUI

**Comment by ischoegl** [comment]
From another user’s perspective, we gave `pycromanager` serious thought but ended up staying with `pymmcore` for the time being. We decided to stick with the older package as access to core functions is straightforward, giving us more flexibility with output formats (`pycromanager`s TIFF format is highly specialized and cannot be read outside of the package; we also didn’t need a GUI). Docs of `pymmcore` may be lacking, but the implementation is relatively intuitive. I assume that `pycromanager` will ultimately get there though.
